### PR TITLE
Add legends in region risk watch

### PR DESCRIPTION
**Pull request by barshathakuri**
## Changes
- Add legends in region risk watch

## This PR doesn't introduce:
- [x] typos
- [x] conflict markers
- [x] unwanted comments
- [x] temporary files, auto-generated files or secret keys
- [x] `console.log` meant for debugging
- [x] codegen errors

### CODE REVIEW

File: src/views/AccountMyFormsDref/ActiveDrefTable/i18n.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
     "namespace": "accountMyFormsDref",
     "strings": {
-        "activeDrefTitle": "Active DREF Operations",
+        "activeDrefTitle": "Completed DREF Operations",
```
Comment (samshara) [inline src/views/AccountMyFormsDref/ActiveDrefTable/i18n.json:4]: Is this the correct string translation?

**Review by samshara — APPROVED** [review]

**Comment by frozenhelium** [comment]
@barshathakuri We should not merge back the `develop` branch to the current branch, we should do the `git rebase develop`

File: src/components/domain/RiskSeasonalMap/i18n.json [hunk 2]
```diff
@@ -9,7 +9,7 @@
         "hazardsTypeLegendLabel": "Type of hazards",
         "peopleExposedOptionLabel": "People Exposed",
         "peopleAtRiskOptionLabel": "People at Risk of Displacement",
-        "riskScoreOptionLabel": "Risk Score",
+        "riskScoreOptionLabel": "INFORM Risk Score",
```
Comment (frozenhelium) [inline src/components/domain/RiskSeasonalMap/i18n.json:12]: We are using both the `INFORM` and `WFP` for this one. So we should keep it as `Risk Score`

File: src/views/CountryRiskWatch/RiskBarChart/WildfireChart/index.tsx [hunk 3]
```diff
@@ -220,6 +220,15 @@ function WildfireChart(props: Props) {
...
             <svg className={styles.svg}>
+                <text
+                    className={styles.yAxisLabel}
+                    textAnchor="middle"
+                    transform={`translate(${(chartMargin.left - CHART_OFFSET) / 3},
+                            ${(chartBounds.height - chartMargin.bottom - chartMargin.top) / 2})
+                            rotate(-90)`}
+                >
+                    {strings.monthlySeverityRating}
+                </text>
```
Comment (frozenhelium) [inline src/views/CountryRiskWatch/RiskBarChart/WildfireChart/index.tsx:231]: We should either create a separate component for this or implement it in the `ChartAxes` component

**Review by frozenhelium — CHANGES_REQUESTED** [review]

**Pull request closed without merging**